### ISSUE TITLE
NativeVideoElementWrapper.js:215 Uncaught (in promise) DOMException 

### ISSUE THREAD

**Issue by sarifconrati** [opentok/opentok-react#58]
This exception is thrown while joining a session.

**Comment by gasi** [comment]
Ditto, we are seeing the same. Is this module part of OpenTok or part of the browser? I believe the error was around `this._domElement.src = this._domElement.src`. Maybe it could be wrapped with `try` / `catch`.

**Comment by leitdeux** [comment]
Yeah, the error seems to be found within the OpenTok Client SDK. I think this is the method of the class `NativeVideoElementWrapper` we're referring to:
```js
// lines 211-216 of NativeVideoElementWrapper.js
async rebind() {
  if (!this._domElement) {
    throw new Error('Can\'t rebind because _domElement no longer exists');
  }

  // Chrome indicates the error occurred on this line
  this._domElement.srcObject = this._domElement.srcObject;
}
```

**Comment by Stewart-Taylor** [comment]
This appears to be fixed today with OpenTok version 1.16.0 I'm no longer able to reproduce this issue anymore.

I'm not seeing any release notes for 1.16.0 however yet
https://support.tokbox.com/hc/en-us/sections/201722216-New-Releases
https://tokbox.com/developer/sdks/js/release-notes.html
